<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>otpsetup</class>
 <widget class="QWidget" name="otpsetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>otpsetup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="welcomeLabel">
       <property name="text">
        <string>Welcome</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>488</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCommandLinkButton" name="welcomeCommandLine">
         <property name="text">
          <string>Welcome</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="releaseCommandLine">
         <property name="text">
          <string>Release information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="installCommandLine">
         <property name="text">
          <string>Install a Product</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="exitCommandLine">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QWebView" name="otpWebView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
